go.mod

```diff
@@ -84,14 +84,16 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.5
 	go.uber.org/atomic v1.10.0
 	go.uber.org/zap v1.23.0
-	golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90
-	golang.org/x/oauth2 v0.0.0-20220909003341-f21342109be1
-	golang.org/x/sys v0.0.0-20220915200043-7b5979e65e41
-	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9
+	golang.org/x/crypto v0.0.0-20221005025214-4161e89ecf1b
+	golang.org/x/oauth2 v0.0.0-20221006150949-b44042a4b9c1
+	golang.org/x/sys v0.0.0-20221006211917-84dc82d7e875
+	golang.org/x/time v0.0.0-20220922220347-f3bd1da661af
 	google.golang.org/api v0.96.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/minio/console => github.com/donatello/console v0.12.6-0.20221013233552-d8ea11bef5c9
+
```

Comment:
Please remove this replace

Reply:
This is the part bringing in the code from PR https://github.com/minio/console/pull/2382 - without it, it won't compile

Reply:
Of course we will remove it when the console PR is merged!